playwright.config.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+const MAX_RETRIES = 3;
+const isCI = process.env.CI === "true";
+
+/** @type { import('@playwright/test').PlaywrightTestConfig} */
+module.exports = {
```

Comment:
Ideally you use [defineConfig](https://github.com/microsoft/create-playwright/blob/11c9c573970ae3c9e1abbf0dcb9d106d2a9729d8/assets/playwright.config.js#L13) + '// @ts-check'